contracts/supplementary/rebalancing-manager/BTCETHRebalancingManager.sol

```diff
@@ -158,6 +172,14 @@ contract BTCETHRebalancingManager {
             auctionStartPrice,
             auctionPivotPrice
         );
+
+        emit LogManagerProposal(
+            auctionStartPrice,
+            auctionPivotPrice,
+            btcPrice,
+            ethPrice,
+            currentSetDollarAmount
```

Comment:
Do we want to log to currentSetDollarAmount or the nextSetDollarAmount (or both?). I'm not sure the currentSetDollarAmount is as relevant.

Reply:
Hmmm do we even need to log the dollar amount? It can be implicitly calculated on the backend with the nextSet address and the prices.